apps/acrs/views/content/referral-submitted.md

```diff
@@ -0,0 +1,10 @@
+We have sent a confirmation email to <strong> {{values.decision-email}} </strong>
+
+<br>
```

Comment:
Don't think the <br> tags are needed in markdown files, unless it's after a bullet point list; blank lines should be sufficient. Try without the tags and see if it looks okay.